eng/Versions.props

```diff
@@ -45,6 +45,8 @@
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</CLI_NETStandardLibraryNETFrameworkVersion>
     <SharedHostVersion>$(MicrosoftNETCoreAppPackageVersion)</SharedHostVersion>
     <HostFxrVersion>$(MicrosoftNETCoreAppPackageVersion)</HostFxrVersion>
+    <!-- <TargetingPackVersion>$(MicrosoftNETCoreAppPackageVersion)</TargetingPackVersion> -->
+    <TargetingPackVersion>3.0.0-preview4-27419-5</TargetingPackVersion>
```

Comment:
Just making a comment here so that we remember to undo this before merging.

Reply:
When this problem first came up in IM, I thought it was because core-sdk didn't branch off for the next preview yet. Now I see this PR is to `release/3.0.1xx` rather than `master`. (My changes are in core-setup `master`, not `release/3.0`.)

I'm familiar with this workflow for 2.1/2.2 fixes (especially infra), didn't expect to see it for brand new stuff like this. Maybe that's an arbitrary distinction, but generally I hear about proving things out in `master` before putting them in release branches. E.g. a new feature like this would be added to `master`, then ported to the oldest applicable release branch, then it would merge its way through to the latest one. I don't know if anyone else has this expectation, though.

Reply:
@dagood - we've just gone through the branching exercise for 3.0 so where things land first is in flux.